tests/conftest.py

```diff
@@ -25,16 +28,18 @@
 API_TOKEN_PATH = os.environ.get("API_TOKEN_PATH")
 API_TOKEN = os.environ.get("API_TOKEN")
 
-# the following five env variables are associated as part of same extraction workflow based on 
+# the following five env variables are associated as part of same extraction workflow based on
 # financial disclosure CSV snapshot and associated workflow
 DATASET_ID = os.environ.get("DATASET_ID")
 WORKFLOW_ID = os.environ.get("WORKFLOW_ID")
+TEACH_TASK_ID = os.environ.get("TEACH_TASK_ID", 1746)
```

Comment:
Can we add this to github secret env variables so build will pass

Reply:
can we close this comment or still relevant?